crates/dapf/src/functions/mod.rs

```diff
@@ -0,0 +1,6 @@
+//! The various DAP functions dapf can perform.
+// Copyright (c) 2024 Cloudflare, Inc. All rights reserved.
+// SPDX-License-Identifier: BSD-3-Clause
+
+pub mod hpke;
+pub mod test_routes;
```

Comment:
What does "test routes" mean in this context? If you're referring to the interop API, then document it here. If these includce non-standard APIs that are specific to Daphne, then document this as well.

Reply:
It maps to routes in the file with the same name `daphne-server/src/router/test_routes.rs`. Do you think we should rename all of these to something that references interop more clearly?

Reply:
Thanks for clarifying. No need to rename things, just add some documentation.